src/CoinbaseWalletSDK.ts

```diff
@@ -184,7 +176,7 @@ export class CoinbaseWalletSDK {
    * Return QR URL for mobile wallet connection, will return null if extension is installed
    */
   public getQrUrl(): string | null {
-    return this._qrUrl;
+    return this._relay?.getQRCodeUrl() ?? null;
```

Comment:
when would `this._relay` not be set? when would `getQRCodeUrl()` return undefined (and why would we need to coerce undefined to null)?

Reply:
`this._relay` would not be set because of [this early return in the constructor](https://github.com/coinbase/coinbase-wallet-sdk/blob/master/src/CoinbaseWalletSDK.ts#L81-L83)

Reply:
since `this._relay` could be `undefined` (because of the early return in the constructor), the expression would resolve to `undefined`, but `getQrUrl` returns `string | null`. So I opted to coerce to `null`. We probably wouldn't want to change the return type of `getQrUrl` at this point

Reply:
ah, i see what you mean, thanks.

Reply:
actually, `this._relay` could not be `undefined`, it could only be `null` or `WalletSDKRelay`. i believe the optional chaining operator is what causes the expression to resolve to `undefined` if it encounters `null`

Reply:
confirmed:
<img width="185" alt="image" src="https://user-images.githubusercontent.com/15928227/163485858-c6f25d31-3161-4616-bafa-84392a363ffb.png">